src/components/form-elements/Input.jsx

```diff
@@ -70,17 +70,17 @@ export const COLOR = Object.freeze({
 });
 
 export type InputPropsType = {
-  type?: ?InputType,
-  value?: string | number,
-  size?: ?InputSizeType,
-  color?: ?InputColorType,
-  valid?: boolean,
-  invalid?: boolean,
-  fullWidth?: boolean,
-  noBorder?: boolean,
-  withIcon?: boolean,
-  className?: string,
-  setInputRef?: (ref: ?HTMLElement) => mixed
+  +type?: ?InputType,
```

Comment:
it will break autocomplete in VSCode. we need to bump to 0.105 before using proper exact/readonly, before that keep objects as it was before

Reply:
updated, damn vscode :(